spec/02-integration/04-admin_api/07-upstreams_routes_spec.lua

```diff
@@ -311,6 +351,68 @@ describe("Admin API: #" .. kong_config.database, function()
             local json = cjson.decode(body)
             assert.same({ message = "Hashing on 'header', but no header name provided" }, json)
 
+            -- Invalid cookie
+            res = assert(client:send {
+              method = "POST",
+              path = "/upstreams",
+              body = {
+                name = "my.upstream",
+                hash_on = "cookie",
+                hash_on_cookie = "not a <> valid <> cookie name",
+              },
+              headers = {["Content-Type"] = content_type}
+            })
+            body = assert.res_status(400, res)
+            local json = cjson.decode(body)
+            assert.same({ message = "Cookie name: bad cookie name 'not a <> valid <> cookie name', allowed characters are A-Z, a-z, 0-9, '_', and '-'" }, json)
+
+            -- Invalid cookie
```

Comment:
duplicate of the one above? maybe this is supposed to be the fallback?

Reply:
Yeah, copy/ paste error. I meant to check validating the cookie path on hash fallback.